charts/gardener/resource-manager/values.yaml

```diff
@@ -13,6 +13,10 @@ global:
   podLabels: {}
   priorityClassName: ""
 
+# nodeToleration:
+#   defaultNotReadyTolerationSeconds: 60
+#   defaultUnreachableTolerationSeconds: 60
```

Comment:
Instead of having separate configuration option for GRM itself, why not just take the same value from the config? This makes it easier for operators to configure this. You also do this for gardenlet already